src/pkg/services/m365/api/events_pager.go

```diff
@@ -244,7 +244,7 @@ func (p *eventDeltaPager) ValidModTimes() bool {
 func (c Events) GetAddedAndRemovedItemIDs(
 	ctx context.Context,
 	userID, containerID, prevDeltaLink string,
-	cc CallConfig,
+	config CallConfig,
```

Comment:
Should we add `LimitResults` and other configs to clues?

Reply:
hmmm let's actually punt that to another PR. I forgot the config isn't passed through to the pagers so either it's going to be some moving structs around or adding lots of individual logging statements. Both of those will generate a fair number of changes so it'd be best to do it separate from these changes

Reply:
created #4681 to track this